build.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: 'idea'
+apply from: 'gradle/quality.gradle'
+
+sourceCompatibility = '1.7'
+targetCompatibility = '1.7'
+
+[compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
+
+mainClassName = "org.apache.mesos.hdfs.Main"
+
+group = "org.apache.mesos"
+version = "0.0.2-SNAPSHOT"
+
+ext {
+    mesosVer = "0.21.1"
```

Comment:
We could update to Mesos 0.22, if we wanted to take advantage of task labels, or service DiscoveryInfo. Would also be nice to point out that we can do disk isolation of HDFS tasks (assuming we actually use the disk resources properly https://github.com/mesosphere/hdfs/issues/77).